cmake/templates/HPXConfig.cmake.in

```diff
@@ -46,22 +63,28 @@ set(HPX_TBBMALLOC_ROOT "@TBBMALLOC_ROOT@")
 # Special handle for mimalloc cause we can't specify HPX_MIMALLOC_ROOT as a HINT
 # to find_package
 set(HPX_MIMALLOC_ROOT "@MIMALLOC_ROOT@")
-if (NOT MIMALLOC_ROOT AND NOT "$ENV{MIMALLOC_ROOT}")
+if(NOT MIMALLOC_ROOT AND NOT "$ENV{MIMALLOC_ROOT}")
   set(MIMALLOC_ROOT ${HPX_MIMALLOC_ROOT})
 endif()
 include(HPX_SetupAllocator)
 
-# Boost
-# Separate boost targets to be unarily linked to some modules
+# Boost Separate boost targets to be unarily linked to some modules
 set(HPX_BOOST_ROOT "@BOOST_ROOT@")
 # By default BOOST_ROOT is set to HPX_BOOST_ROOT (not necessary for PAPI or
 # HWLOC cause we are specifying HPX_<lib>_ROOT as an HINT to find_package)
-if (NOT BOOST_ROOT AND NOT "$ENV{BOOST_ROOT}")
+if(NOT BOOST_ROOT AND NOT "$ENV{BOOST_ROOT}")
   set(BOOST_ROOT ${HPX_BOOST_ROOT})
 endif()
 include(HPX_SetupBoost)
 include(HPX_SetupBoostFilesystem)
 include(HPX_SetupBoostProgramOptions)
+include(HPX_SetupBoostProgramOptions)
```

Comment:
@msimberg This looks like a copy&paste mishap.

Reply:
Looks like I left this one in. I'll remove this directly on master.